### PR TITLE
Making changes suggested by our docs writers to elasticsearch and mysql sed

### PR DESCRIPTION
**Pull request by Kartikey-star**
Signed-off-by: Kartikey Mamgain <mamgainkartikey@gmail.com>

### CODE REVIEW

File: seds/elasticsearch-sed/README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-This helm chart defines a Elasticsearch Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This helm chart defines a Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Comment (Srivaralakshmi) [inline seds/elasticsearch-sed/README.md]:
```suggestion
This Helm chart defines an Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Suggestion: The H in Helm is always in capitals. Global comment. Please change in all such instances.

File: seds/elasticsearch-sed/README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 1. Hostname
 2. Port
 3. User
 4. Password
 5. Cluster
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by Elasticsearch SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding Specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s[Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/elasticsearch-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Suggestion: Apply the comment to MySQL SED content also.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@Kartikey-star Left some suggestions. PTAL. Thanks!

File: seds/elasticsearch-sed/README.md (hunk 1 above)
Comment (Kartikey-star) [reply seds/elasticsearch-sed/README.md]: Made the change

File: seds/elasticsearch-sed/README.md (hunk 2 above)
Comment (Kartikey-star) [reply seds/elasticsearch-sed/README.md]: Made the change

File: seds/elasticsearch-sed/README.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 
 1. Hostname
 2. Port
 3. User
 4. Password
 5. Cluster
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by Elasticsearch SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s[Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/elasticsearch-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```

File: seds/mysql-sed/README.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
...
 
 1. Host
 1. Port
 1. Username
 1. Password
 1. Databasename
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by MySQL SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
\ No newline at end of file
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MySQL SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Binding Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/mysql-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification’s [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the MySQL SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```

File: seds/mysql-sed/README.md [hunk 5]
```diff
@@ -1,9 +1,9 @@
-This helm chart defines a MySQL Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This Helm chart defines a MySQL Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Comment (Srivaralakshmi) [inline seds/mysql-sed/README.md]:
```suggestion
This Helm chart defines a MySQL Service Endpoint Definition (SED). When the SED chart is installed, it provides the user with the opportunity to provide binding data and credentials to authenticate. The following values must be customized while installing the SED chart:
```

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@Kartikey-star Left some comments. PTAL. Thanks!

File: seds/elasticsearch-sed/README.md [hunk 6]
```diff
@@ -1,9 +1,9 @@
-This helm chart defines a Elasticsearch Service Endpoint Definition (SED). When the SED is installed it will provide the user with the oportunity to provide connection information as well as credentials to authenticate. The following are the values that can be customized when the SED chart is installed:
+This Helm chart defines an Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed it provides the user with the opportunity to provide binding data as well as credentials to authenticate. The following values are to be customized while installing the SED chart:
```
Comment (Srivaralakshmi) [inline seds/elasticsearch-sed/README.md]:
```suggestion
This Helm chart defines an Elasticsearch Service Endpoint Definition (SED). When the SED chart is installed, it provides the user with the opportunity to provide binding data and credentials to authenticate. The following values must be customized while installing the SED chart:
```

File: seds/mysql-sed/README.md (hunk 5 above)
Comment (Kartikey-star) [reply seds/mysql-sed/README.md]: Changes Done

File: seds/elasticsearch-sed/README.md (hunk 6 above)
Comment (Kartikey-star) [reply seds/elasticsearch-sed/README.md]: Changes made

File: seds/elasticsearch-sed/README.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
...
 
 1. Hostname
 2. Port
 3. User
 4. Password
 5. Cluster
 
-The SED Chart will render a secret with the connection information. This secret is compliant with the Service Binding Specification [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by Elasticsearch SED Chart is a bindable service endpoint that can be projected to workloads using the Service [Binding Direct Secret Reference](https://github.com/servicebinding/spec#well-known-secret-entries).
+The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s[Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```
Comment (Srivaralakshmi) [inline seds/elasticsearch-sed/README.md]:
```suggestion
The SED chart will render a secret with the binding data. This secret is compliant with the Service Binding specification's [Well Known Secret Entries](https://github.com/servicebinding/spec#well-known-secret-entries). Therefore, the secret rendered by the Elasticsearch SED chart is a bindable service endpoint that can be projected to workloads using the Service Binding specification’s [Direct Secret Reference](https://github.com/servicebinding/spec#direct-secret-reference).
```

Comment (Srivaralakshmi) [reply seds/elasticsearch-sed/README.md]: Added space. Please apply this change.

**Review by Srivaralakshmi — CHANGES_REQUESTED** [review]
@Kartikey-star Seems you have missed out on this comment. PTAL. Thanks!

File: seds/elasticsearch-sed/README.md (hunk 7 above)
Comment (Kartikey-star) [reply seds/elasticsearch-sed/README.md]: Made the change

**Review by Srivaralakshmi — APPROVED** [review]
@Kartikey-star Approved! Thanks for the great work!

**Pull request merged**